nginx/proxy.template

```diff
@@ -0,0 +1,69 @@
+server {
```

Comment:
How this file was derived:
1. copy https template
2. Remove all SSL bits
3. Use http port